docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,155 @@
+# Tizen Window System Shell
+
+
+The TZSH is an interface for manipulating windows of the sytem GUI services such as the quickpanel and the softkey. It provides C libraries that allow you to control windows of system GUI services and get notifications about the state changes of each window within an application.
+
+Since, the system GUI service runs different process, all of operations of the TZSH are performed using the inter-process communication (IPC). It is needed to enable communication between system GUI service processes and separated user application process using the display server. The following figure illustrates a simple software architecture of the TZSH:
+
+**Figure: Layer diagram for the Tizen window system shell**
+
+![Layer diagram for the Tizen window system shell](./media/tzsh_arch.png)
+
+To support each system GUI service in Tizen, the TZSH provides the following libraries:
+
+- **tzsh-quickpanel**: For the quickpanel window which shows notifications and system setup widgets.
+
+- **tzsh-softkey**: For the softkey window which shows the device back and home buttons.
+
+Not all applications require to use the TZSH's functionalities. However, in some cases, certain applications may require to perform manipulation of system GUI service's window. For example, media player application needs to close the quickpanel window during playback of video. In this case, you can use the tzsh-quickpanel library.
+
+
+## Prerequisites
+
+To use the functions and structures of the tzsh-quickpanel API (in [mobile](../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) and [wearable](../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) applications), include the `<tzsh_quickpanel.h>` header file in your application:
```

Comment:
tzsh-quickpanel API -> Tizen WS Shell Quickpanel API

Please use the word used in API reference file.

**Just a question** to @gl-lee :
In the native/guides/ui/overview.md's update and the starting of this tizen-ws-shell.md page, 
TZSH(about quickpanel and softkey) is described. 
However from here, this page describes quickpanel only .

Are there other pages to be added under Tizen WS Shell, as "Tizen WS Shell Quickpanel"?
Or is the "Tizen WS Shell Quickpanel" only page for Tizen WS Shell?
For now this page seems mixed version of " Tizen WS Shell" and "Tizen WS Shell Quickpanel" to me.

**Another question I got during API Reference checking:**
Isn't this update for 5.5, which should be managed on `tizen_5.5_prepare` branch?
`master` branch is to handle documents for tizen 5.0, which will be published after merging.

Reply:
I've changed 'tzsh-quickpanel API' to 'TZSH quickpanel API'.

And TZSH has a feature of the softkey, but ACR for the sofkey is now in progress. When ACR of the softkey proceeds to the guide document stage, then I will add a content about the softkey to this TZSH page.

Reply:
I was guided by the master branch for reviewing, and I heard that I will apply tizen_5.5_prepare branch after reviewing. Could you let me know exact order?

Reply:
I checked with @journey2w, and found out that this module is already included in Tizen 5.0.
So, your way now is correct.

Thanks for comment and please keep processing this PR.

Reply:
@mijin85cho  & @gl-lee  thanks for your time and efforts.

Reply:
Very thanks @nawab-samsung & @mijin85cho. :)